src/qt/transactiondesc.cpp

```diff
@@ -217,6 +268,8 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, int vout, int u
 
     strHTML += "<b>" + tr("Net amount") + ":</b> " + BitcoinUnits::formatWithUnit(unit, nNet, true) + "<br>";
 
+    strHTML += "<b>" + tr("Data") + ":</b> " + dataStr + "<br>";
```

Comment:
``dataStr`` should be escaped, since the contents could be HTML. There is examples of using ``GUIUtil::HtmlEscape`` elsewhere in this function that should do what's needed.